templates/test/ci/prow-dalec-custom-builds/patches/control-plane-custom-builds.yaml

```diff
@@ -0,0 +1,46 @@
+- op: add
+  path: /spec/kubeadmConfigSpec/files/-
+  value:
+    content: |
+      #!/bin/bash
+      set -o nounset
+      set -o pipefail
+      set -o errexit
+      systemctl stop kubelet
+      declare -a BINARIES=("kubeadm" "kubectl" "kubelet")
+      # Define the base URL and version
+      BASE_URL="https://kubernetesreleases.blob.core.windows.net/dalec-packages"
+      VERSION="${KUBERNETES_VERSION}"
+      VERSION=$${VERSION#v}
+      OS_VERSION="ubuntu24.04"
+      ARCH="amd64"
+      for BINARY in "$${BINARIES[@]}"; do
+        if [[ "$${BINARY}" == "kubelet" ]]; then
+          DEB_VERSION="ubuntu24.04u1"
```

Comment:
I'm a bit confused: so this PR also adds a dalec-custom-builds template, but it's *not* using Azure Linux?